backend/src/main/resources/graphql/admin.graphqls

```diff
@@ -10,15 +10,16 @@ extend type Mutation {
   createDeviceType(input: CreateDeviceType!): DeviceType
   updateDeviceType(input: UpdateDeviceType!): DeviceType
   createSpecimenType(input: CreateSpecimenType!): SpecimenType
-  addUser( # not actually used in the admin console (currently)
+  addUser( # used in AddOrganizationAdminFormContainer
     name: NameInput
```

Comment:
do we want to add the deprecated message here too

Reply:
yeah! thanks for catching it